scripts/kmodtool

```diff
@@ -144,6 +144,12 @@ print_rpmtemplate_per_kmodpkg ()
 	local kernel_uname_r=${1}
 	local kernel_variant="${2:+-${2}}"
 
+	# With RHEL 8, depmod moved from /sbin to /usr/sbin
```

Comment:
I find that in RHEL7.7 and RHEL8, they both symlink to ../bin/kmod
lrwxrwxrwx. 1 root root 11 Feb 10  2019 /sbin/depmod -> ../bin/kmod
lrwxrwxrwx. 1 root root 11 Feb 10  2019 /usr/sbin/depmod -> ../bin/kmod

Reply:
That's right.  Either path can be used to execute `depmod` during the install.  The issue being addressed here is in the RPM packaging where the real path, and not a symlink, must be specify to set the correct dependency.